Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -4049,10 +4049,6 @@ AccessibilityObjectInclusion AccessibilityObject::defaultObjectInclusion() const
         if (style->effectiveInert())
             return AccessibilityObjectInclusion::IgnoreObject;
         if (style->usedVisibility() != Visibility::Visible) {
-            // aria-hidden is meant to override visibility as the determinant in AX hierarchy inclusion.
-            if (equalLettersIgnoringASCIICase(getAttribute(aria_hiddenAttr), "false"_s))
-                return AccessibilityObjectInclusion::DefaultBehavior;
-
             return AccessibilityObjectInclusion::IgnoreObject;
         }
```

Comment:
The curly-braces here are unnecessary now.

Reply:
Thanks for catching that—surprising that the styler checker didn't call that out